README.md

```diff
@@ -116,15 +138,25 @@ Build with the usual
 
     docker build -t maven .
 
-Tests are written using [bats](https://github.com/sstephenson/bats) under the `tests` dir.
+Tests are written using [bats](https://github.com/sstephenson/bats) for Linux images and [pester](https://github.com/pester/Pester) for Windows images 
+(requires Pester 4.x) under the `tests` dir.
+
 Use the env var TAG to choose what image to run tests against.
 
+### Linux
     TAG=jdk-11 bats tests
 
+### Windows
+    $env:TAG="jdk-11" ; Invoke-Pester -Path tests
```

Comment:
Again, if this is Powershell, would be polite to call that out at least with syntax highlighting.